client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -296,16 +295,17 @@ import {
   VPopover
 } from '@demos-europe/demosplan-ui'
 import { mapActions, mapMutations, mapState } from 'vuex'
+import DpAddonWrapper from '../../../../../addons-bkp/cache/demosplan-addon-demospipes-f_T29585_similar_segments_ui/client/DpAddonWrapper.vue'
```

Comment:
this looks wrong.
when we move the addonWrapper to dp-ui, we have to be able to use `import DpAddonWrapper from "@demos-europe/demosplan-ui"` here too

Reply:
adjusted these dev leftovers here: https://github.com/demos-europe/demosplan-core/pull/1486/commits/d149bc96ccca6d37cd5c0e6ed000903b42e9d0d4

Also re-added DpInsertableRecommendations.vue here: https://github.com/demos-europe/demosplan-core/pull/1486/commits/cf1105528444515355d7e72e56542290400514a2